src/utils/update/company.profile.mapper.to.oversea.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { Entity } from "../../model/entity.model";
+
+export const mapCompanyProfileToOverseasEntityToDTOUpdate = (cp: CompanyProfile): Entity => {
+  if (!cp){
```

Comment:
We should not mask the failure of calling this with null or undefined. It will only lead to problems later.

Reply:
See also comment on not calling this when you don't have a company_profile result.

Reply:
👍🏿